packages/dx-react-grid-bootstrap4/src/templates/table-tree-toggle-button.jsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import { ToggleButton } from './parts/toggle-button';
+
+export const TableTreeToggleButton = ({
```

Comment:
Personally, I don't like the 'TableTree...' words combination. Maybe we can think of some better option.

Reply:
This is internal name. It should be specific enough as it present in a global namespace.